modules/nf-core/vcflib/vcfwave/meta.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: "vcflib_vcfwave"
+description: write your description here
```

Comment:
Please do 😛

Reply:
Gotta get this stupid module working first :grin: